### PR TITLE
qa/tests: run ceph-ansible task on installer.0 role/node

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

**Comment by yuriw** [comment]
@vasukulkarni @wusui does it look reasonable ?  I am not sure about `4-tasks/rest.yaml` is it needs to change?

File: qa/suites/ceph-ansible/smoke/basic/0-clusters/3-node.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ meta:
 - desc: "3-node cluster"
 roles:
 - [mon.a, mds.a, osd.0, osd.1, osd.2]
-- [mon.b, mgr.x, osd.3, osd.4, osd.5]
-- [mon.c, mgr.y, osd.6, osd.7, osd.8, client.0]
+- [mon.b, mgr.x, osd.3, osd.4, osd.5, mon.c, mgr.y, osd.6, osd.7, osd.8, client.0]
+- [installer.0]
```
Comment (vasukulkarni) [inline qa/suites/ceph-ansible/smoke/basic/0-clusters/3-node.yaml]: Just add installer.0, we need mon.c to be on another node

File: qa/suites/ceph-ansible/smoke/basic/4-tasks/ceph-admin-commands.yaml [hunk 2]
```diff
@@ -3,5 +3,5 @@ meta:
 tasks:
 - workunit:
     clients:
-      client.0:
+      installer.0:
```
Comment (vasukulkarni) [inline qa/suites/ceph-ansible/smoke/basic/4-tasks/ceph-admin-commands.yaml]: should be left as is to original, installer is only for ceph-ansible node

File: qa/suites/ceph-ansible/smoke/basic/4-tasks/rbd_import_export.yaml [hunk 3]
```diff
@@ -3,5 +3,5 @@ meta:
 tasks:
 - workunit:
     clients:
-      client.0:
+      installer.0:
```
Comment (vasukulkarni) [inline qa/suites/ceph-ansible/smoke/basic/4-tasks/rbd_import_export.yaml]: No need for this change as commented earlier.

Comment (yuriw) [reply qa/suites/ceph-ansible/smoke/basic/4-tasks/rbd_import_export.yaml]: how does`ceph-ansible` know to run on `installer.0` ?

**Comment by yuriw** [comment]
testing http://pulpito.ceph.com/teuthology-2017-12-19_21:13:39-ceph-ansible-master-distro-basic-vps/

**Comment by vasukulkarni** [comment]
@yuriw also commit needs change to something like:  "qa/tests: use installer role for ceph-ansible installation"?

**Comment by yuriw** [comment]
And passed http://pulpito.ceph.com/teuthology-2017-12-22_16:56:47-ceph-ansible-master-distro-basic-ovh/ on a fix https://github.com/ceph/ceph/pull/19603

**Comment by yuriw** [comment]
jenkins test docs please

**Pull request merged**